docs/0-installation.md

```diff
@@ -48,14 +48,46 @@ The generator adds these core files, among others:
 * `app/assets/stylesheets/active_admin.scss`
 * `config/initializers/active_admin.rb`
 
-Now, migrate and seed your database before starting the server:
+Now, migrate and seed your database:
 
 ```sh
 rails db:migrate
 rails db:seed
-rails server
 ```
 
+These instructions suggest using the `cssbundling-rails` and `importmap-rails`
+gems for a streamlined asset management setup. However, this is not mandatory,
+and you can handle asset bundling in any way that suits your needs, as long as
+it works effectively.
+
+If you haven't installed and configured `cssbundling-rails` and
+`importmap-rails`, please run:
+
+  ```sh
+  bundle add importmap-rails
+  rails importmap:install
+
+  bundle add cssbundling-rails
+  rails css:install:tailwind
```

Comment:
It's not necessary to install tailwind. One can run command `rails css:install` and it should work.

Reply:
I mean, gem 'tailwindcss-rails' may not be installed